packages/components/src/utils/link-button/link-button-variant.ts

```diff
@@ -1,2 +1,2 @@
-export const LINK_BUTTON_VARIANTS = ['primary', 'secondary', 'tertiary'] as const;
-export type LinkButtonVariant = typeof LINK_BUTTON_VARIANTS[number];
+export const LINK_BUTTON_VARIANTS = ['primary', 'secondary', 'tertiary', 'ghost'] as const;
```

Comment:
could be approved with LINK_BUTTON_VARIANTS_DEPRECATED + types + docs, etc.

Reply:
You mean like this?

```
/** @deprecated */
export const LINK_BUTTON_VARIANTS_DEPRECATED = ['tertiary'] as const;
/** @deprecated */
export type LinkButtonVariantDeprecated = (typeof LINK_BUTTON_VARIANTS_DEPRECATED)[number];

export const LINK_BUTTON_VARIANTS = ['primary', 'secondary', 'ghost', ...LINK_BUTTON_VARIANTS_DEPRECATED] as const;
export type LinkButtonVariant = (typeof LINK_BUTTON_VARIANTS)[number];
```